src/cobra/io/mat.py

```diff
@@ -681,6 +684,11 @@ def from_mat_struct(
         met_comps = [comps[i] for i in met_comp_index]
         met_comp_names = [comp_names[i] for i in met_comp_index]
     else:
+        logger.warning(
```

Comment:
I think it would be cool if that warning also reports the names of all unique compartments. That way one can see immediately if there are wonky ones.

Reply:
Like this new commit?

Reply:
Done